components/EditCarForm/index.js

```diff
@@ -71,6 +71,9 @@ export default function EditCarForm({ onSubmit, initialValues }) {
                       {attribute}:
                       {type === "text" ? (
                         <input
+                          required={
+                            attribute === "Make" || "Model" ? true : false
+                          }
```

Comment:
can be shortened since comparing two values always evaluates to a boolean value

```suggestion
                          required={attribute === "Make" || "Model"}
```

Reply:
Thank you, I implemented it